From 035db31ae318955653dc54fc279b56bd4f9cacc1 Mon Sep 17 00:00:00 2001
From: Igor Grinberg <igrinberg@habana.ai>
Date: Sun, 07 Apr 2019 13:14:25 +0300
Subject: [PATCH] [SW-0] goya: pll: raise CPU frequency

Raise the CPU frequency to achieve a better AXI and DMA performance.
The new CPU frequency is 1333MHz.

Change-Id: I13bc1b04a2c2c9e075ee96af22095ec31482f0ce
Signed-off-by: Igor Grinberg <igrinberg@habana.ai>
---

diff --git a/goya/pll.c b/goya/pll.c
index 0449ea9..df855cf 100644
--- a/goya/pll.c
+++ b/goya/pll.c
@@ -59,7 +59,7 @@
 static struct hl_pll_cfg goya_pll_cfg[] = {
 	{
 		.name = "CPU", .base = (void *)mmCPU_PLL_BASE,
-		.nr = 0, .nf = 47, .od = 1, .nb = 23,
+		.nr = 2, .nf = 159, .od = 1, .nb = 79,
 		.div_factor = { 0, -1, 1, -1 }
 	}, {
 		.name = "IC", .base = (void *)mmIC_PLL_BASE,
